misc/dbt-materialize/tests/adapter/test_custom_materializations.py

```diff
@@ -46,30 +48,35 @@ def models(self):
         return {
             "actual_indexes.sql": actual_indexes,
             "test_materialized_view.sql": test_materialized_view,
+            "test_materialized_view_grant.sql": test_materialized_view_grant,
             "test_materialized_view_index.sql": test_materialized_view_index,
             "test_relation_name_loooooooooooooooooonger_than_postgres_63_limit.sql": test_relation_name_length,
             "test_source.sql": test_source,
             "test_source_index.sql": test_source_index,
             "test_subsources.sql": test_subsources,
             "test_sink.sql": test_sink,
             "test_table_index.sql": test_table_index,
+            "test_view_grant.sql": test_view_grant,
             "test_view_index.sql": test_view_index,
         }
 
     def test_custom_materializations(self, project):
+        project.run_sql("CREATE ROLE bi")
+        project.run_sql("CREATE ROLE reporter")
```

Comment:
A little stumped on the best way to configure testing grants. This hook is successful with my staging environment but getting a generic `Database Error in model` with the unit tests. Not sure what additional grants or configs I should be setting to allow this to run.

Reply:
Ugh. It took me a bit to decipher this. Still have not a full clear idea but the following query is run by one of the tests:

```sql
select grantee, privilege_type
  from "materialize".INFORMATION_SCHEMA.role_table_grants
      where grantor = current_role
        and grantee != current_role
        and table_schema = 'test16963496672607561145_test_custom_materializations'
        and table_name = 'test_materialized_view_grant';
```

This query makes Materialize crash. Then (my guess) is that the rest of the tests fail because they can't connect to Materialize or the state is wrong. We should report it, if it hasn't already been reported.